packages/xrpl/test/client.test.ts

```diff
@@ -0,0 +1,53 @@
+import { assert } from 'chai'
```

Comment:
Can the renames to `.test.ts` be done in a separate PR? Most of these diffs are impossible to review because git treats them as separate files.

Reply:
https://github.com/XRPLF/xrpl.js/pull/2181